src/App.tsx

```diff
@@ -96,8 +98,14 @@ const App = () => {
             <Route path='feed' element={<FeedClaim {...commonProps} />} />
             <Route path='search' element={<Search {...commonProps} />} />
             <Route path='/' element={<Form {...commonProps} />} />
-            <Route path='login' element={<Login {...commonProps} />} />
             <Route path='register' element={<Register {...commonProps} />} />
+            <Route path='login' element={<Login {...commonProps} />} />
+            <Route
+              path='/rate'
```

Comment:
Make a config array and map it into route

Reply:
hey @sahdasamier @Zakiamangal you guys could do this suggestion later?